### ISSUE TITLE
std::thread support

### ISSUE THREAD

**Issue by nipunG314** [electronicarts/EASTL#528]
Is there any plan to add support for the C++11 std::thread api in the future?

**Comment by jhopkins-ea** [comment]
No. EASTL is a library of containers and related functionality (eg. algorithms) and isn't a full replacement of the C++ standard library.
